assets/js/home.js

```diff
@@ -32,10 +32,18 @@ $(document).ready(function () {
     }
     $('#log').prepend($.markup('log-entry', data))
   }
+  $.get('/updated', function (response) {
```

Comment:
There is no need to call this here, since it is already included in the `refreshData` function, which is automatically called once the markup is initialised